NYUDining/FbLoginViewController.swift

```diff
@@ -0,0 +1,86 @@
+
+//
+//  FbLoginViewController.swift
+//  NYUDining
+//
+//  Created by Jesus Leal on 6/29/16.
+//  Copyright © 2016 Ross Freeman. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class FbLoginViewController: UIViewController, FBSDKLoginButtonDelegate
+{
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+        //configureFacebook()
+        /*if (FBSDKAccessToken.currentAccessToken() != nil)
+        {
+            let loginManager: FBSDKLoginManager = FBSDKLoginManager()
+            loginManager.logOut()
+            FBSDKAccessToken.// User is already logged in, do work such as go to next view controller.
+        }
+        else
+        {*/
+            let loginView : FBSDKLoginButton = FBSDKLoginButton()
+            self.view.addSubview(loginView)
+            loginView.center = self.view.center
+            loginView.readPermissions = ["public_profile", "email", "user_friends"]
+            loginView.delegate = self
+        //}
+    }
+    
+    func loginButton(loginButton: FBSDKLoginButton!, didCompleteWithResult result: FBSDKLoginManagerLoginResult!, error: NSError!)
+    {
+        FBSDKGraphRequest.init(graphPath: "me", parameters: ["fields":"first_name, last_name, picture.type(large)"]).startWithCompletionHandler { (connection, result, error) -> Void in
+            if (result != nil)
+            {
+                let fbUserId: String = (result.objectForKey("id") as? String)!
+                print("\(fbUserId)")
+                let strFirstName: String = (result.objectForKey("first_name") as? String)!
```

Comment:
This logic should are rewritten, probably by using SwiftyJSON. The main problem now is that you're force unwrapping everything. While it isn't a problem assuming the JSON file is always 100% perfect, it's still never a good idea to force unwrap - that's the cause for like 90% of crashes I've experienced.